client/src/components/course-view/CreateAplusAttainments.tsx

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: 2024 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import {Box, Card, CardContent, TextField} from '@mui/material';
+
+import {AplusGradeSourceData} from '@/common/types';
+
+type AttainmentCardProps = {
+  attainment: {name: string; daysValid: number};
+  handleChange: (attainment: {name?: string; daysValid?: number}) => void;
+};
+
+const AttainmentCard = ({
+  attainment,
+  handleChange,
+}: AttainmentCardProps): JSX.Element => {
```

Comment:
Also since this just returns some JSX the return is not necessary, it can just be `...): JSX.Element => ( <Card>...`

Reply:
Unless more functionality is planned soon, but even then it can just be changed back

Reply:
> Also since this just returns some JSX the return is not necessary, it can just be `...): JSX.Element => ( <Card>...`

You think we should prefer that style? We also have some other components (`shared/PrettyChip.tsx`, `auth/ExternalAuth.tsx`) which just return JSX but use a return statement.